README.md

```diff
@@ -55,7 +55,7 @@ Please follow the below instructions to test API endpoints.
 
 Install Python requirements with additional packages for testing:
 ```
-pip install -r docker/api/requirements-dev.txt
```

Comment:
I don't think this should change, the -dev file has all DW packages for development.

Reply:
Testing packages are in `requirements-tests.txt`.

Reply:
Yes but the standard way to install development packages is to install `requirements-dev.txt`.

Reply:
Yes, I dropped the commit.